src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+body,
+.nav {
   margin: 0;
+  padding: 0;
+  max-width: 1200px;
```

Comment:
Remove this for fix comments
```suggestion
```

Reply:
<img width="959" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/129754902/8f85edd7-97d4-4795-a218-aad59195e748">